packages/core/admin/ee/server/services/__tests__/stages.test.js

```diff
@@ -69,6 +69,8 @@ const servicesMock = {
   },
 };
 
+const queryUpdateMock = jest.fn(() => true);
```

Comment:
imo this tests could be integration tests:
- changing how we update the entities should not require you to update tests. 
- We are not really testing if entities stages are really updated on db, I had a bug I noticed when trying it by hand and the unit test was passing.